src/modules/trade/containers/TradeBasicConfirmDetails/MinReceivedRow.tsx

```diff
@@ -2,28 +2,13 @@ import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 
 import { Nullish } from 'types'
 
-import { ConfirmDetailsItem } from 'modules/trade/pure/ConfirmDetailsItem'
-
-import { FiatAmount } from 'common/pure/FiatAmount'
-import { TokenAmount } from 'common/pure/TokenAmount'
+import { ReviewOrderModalAmountRow } from '../../pure/ReviewOrderModalAmountRow'
 
 type Props = {
   amount: Nullish<CurrencyAmount<Currency>>
   usdAmount: Nullish<CurrencyAmount<Currency>>
 }
 
 export function MinReceivedRow(props: Props) {
```

Comment:
Now this component looks excessive, can we directly use `ReviewOrderModalAmountRow`?